__tests__/index.test.js

```diff
@@ -1,21 +1,23 @@
-const devIndex = require('../db/data/development-data/index')
-const testIndex = require('../db/data/test-data/index')
-describe('Development data index has all required keys', () => {
-    test('should come back with correct keys for development index', ()=>{
-        //Arrange
-        //Act
-        //Assertion
-        expect(Object.keys(devIndex)).toEqual(["articleData", "commentData", "topicData", "userData"]);
-        expect().toEqual();
-    });
+const devIndex = require("../db/data/development-data/index");
+const testIndex = require("../db/data/test-data/index");
+describe("Development data index has all required keys", () => {
+  test("should come back with correct keys for development index", () => {
+    expect(Object.keys(devIndex)).toEqual([
+      "articleData",
+      "commentData",
+      "topicData",
+      "userData",
+    ]);
+  });
 });
```

Comment:
Just a little tip, this test isn't needed. The seed function will deal with any issues with the test data if there is any.